### ISSUE TITLE
Issue with I2C variables in PicoAirQuality.py

### ISSUE THREAD

**Issue by cleminan** [KitronikLtd/Kitronik-Pico-Smart-Air-Quality-Board-MicroPython#6]
The current version of PicoAirQuality.py  fails with the following error
```
Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
  File "PicoAirQuality.py", line 562, in __init__
TypeError: extra keyword arguments given
```

The line referenced has a `timeout` value not supported by `I2C`
```python
self.i2c = I2C(1, sda=sda, scl=scl, freq=100_000, timeout=100_000)
```

Changing the class to `SoftI2C` which does support `timeout` doesn't fix the error, but removing the last variable for `I2C` fixes the issue.
```python
self.i2c = I2C(1, sda=sda, scl=scl, freq=100_000)
```

The same issue exists around line 1102.

**Comment by JackAtKitronik** [comment]
Hi, can you tell us which version of MicroPython you are using?

We recently updated the library to fix a I2C timeout issue in MicroPython V1.20.

Thanks,
Jack

**Comment by cleminan** [comment]
Hi,

Thonny installed v1.19.1, I'm just finding my feet so just hit the update process on it when I first connected the Pico.

I'm also having problems getting the ZipLEDs to work, while they don't respond to the `zipleds.SetLED()` call, LEDs 0 and 1 light up when running the Analogue IO Control Tutorial script. But that's another issue & I'm trying to make sure it isn't my fault.

Cheers
a.

**Comment by JackAtKitronik** [comment]
Hi,

I should have fixed the timeout issue for both version 1.19.1 and 1.20.

If you take a copy of the new version of the library everything should work now.

With the ZIP LEDs make sure you are using 0 to 2 with `zipleds.setLED(0, (50, 50, 50))` and remeber to call `zipleds.show()` to update the LEDs after setting them.

Thanks,
Jack
